.github/workflows/cws-btfhub-sync.yml

```diff
@@ -85,5 +85,5 @@ jobs:
               owner,
               repo,
               issue_number: result.data.number,
-              labels: ['changelog/no-changelog', 'qa/skip-qa', 'team/agent-security']
+              labels: ['changelog/no-changelog', 'qa/skip-qa', 'qa/no-code-change', 'team/agent-security']
```

Comment:
Here as well it's probably better to consider this as a `no-code-change`

Reply:
yeah it's fine like this ! the generated PR only touches an embedded json file